src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -35,10 +35,10 @@ angular
  * There is an example below of how this should look.
  *
  * ### Notes
- * The `md-autocomplete` uses the the [VirtualRepeat](/api/directive/mdVirtualRepeatContainer)
+ * The `md-autocomplete` uses the the [VirtualRepeat](api/directive/mdVirtualRepeat)
```

Comment:
This is intentionally linked to the `mdVirtualRepeatContainer`. 
> The label was just Virtual Repeat because the container is actually the over-all component.